app/scripts/background.js

```diff
@@ -448,7 +448,7 @@ function setupController (initState, initLangCode) {
 function triggerUi () {
   extension.tabs.query({ active: true }, tabs => {
     const currentlyActiveMetamaskTab = Boolean(tabs.find(tab => openMetamaskTabsIDs[tab.id]))
-    if ((!popupIsOpen || !notificationIsOpen) && !currentlyActiveMetamaskTab) {
+    if ((!popupIsOpen || notificationIsOpen) && !currentlyActiveMetamaskTab) {
```

Comment:
I think this will result in multiple notification windows being opened if multiple txs are created before any one is confirmed/rejected.

Reply:
That's not the case.

![mar-04-2019 08-29-53](https://user-images.githubusercontent.com/13376180/53739630-d51c7680-3e57-11e9-9c6d-1624b2407625.gif)

Reply:
So maybe `notifcationIsOpen` can be removed entirely from the `if` expression?